tests/puppeteer/campaigns/functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/general/01_enableFinalSummary.js

```diff
@@ -74,17 +74,23 @@ describe('Enable final summary', async () => {
       await expect(result).to.contains(this.pageObjects.orderSettingsPage.successfulUpdateMessage);
     });
 
+    it('should view my shop', async function () {
+      await testContext.addContextItem(this, 'testIdentifier',`${test.args.action}AndViewMyShop`, baseContext);
+      // Click on view my shop
+      page = await this.pageObjects.orderSettingsPage.viewMyShop();
+      this.pageObjects = await init();
+      await this.pageObjects.homePage.changeLanguage('en');
+      const isHomePage = await this.pageObjects.homePage.isHomePage();
+      await expect(isHomePage, 'Home page is not displayed').to.be.true;
+    });
+
     it('should go to FO and check the final summary after checkout', async function () {
```

Comment:
```suggestion
    it('should check the final summary after checkout', async function () {
```
since you're already on the FO thanks to your last step :D

Reply:
done